src/thermo/WaterPropsIAPWS.cpp

```diff
@@ -28,7 +28,7 @@ static const doublereal M_water = 18.015268;
 
 static const double R_water = 461.51805; // J/kg/K (Eq. 6.3)
 
-//! Gas constant that is quoted in the paper
+// Gas constant that is quoted in the paper
```

Comment:
If the above constants (`T_c` etc.) get Doxygen comments, why shouldn't this one as well?